### PR TITLE
Implementation of requests to get debates from an admin and questions from a debate

### LINKED ISSUE #30: Get Discussion created by user

### LINKED ISSUE #31: Get Question from a Discussion
This will also include the response related to the Question

### PR DESCRIPTION
**Pull request by Naludrag**
closes #30 
closes #31

### CODE REVIEW

File: src/database/DatabaseManager.js [hunk 1]
```diff
@@ -55,6 +56,56 @@ export class DataBaseManager {
...
+                else{
+                    console.log(discussions);
+                }
+            });
+        }
+        return discussions;
+    }
+
+    /**
+     * Get the questions from disucssion
```
Comment (ghost) [inline src/database/DatabaseManager.js]:
```suggestion
     * Get the questions from discussion
```

File: test/databaseManager.js [hunk 2]
```diff
@@ -1,24 +1,70 @@
...
+        const question2 = new Question({
+            _id: 2,
+            titreQuestion: "Question 2 for debate 1",
+            numberVotes: 0,
+            refDiscussion: discussion1._id
+        });
+        await question2.save().then((questionSaved) => {
+            console.log("question saved : ", questionSaved);
         });
+
```
Comment (ghost) [inline test/databaseManager.js]: Empty line -> remove it  later on please

**Review by ghost — APPROVED** [review]
Before full approval of merging, there is two smalls corrections to apply.

File: test/databaseManager.js (hunk 2 above)
Comment (Naludrag) [reply test/databaseManager.js]: Change with the commit 37bf45f

File: src/database/DatabaseManager.js (hunk 1 above)
Comment (Naludrag) [reply src/database/DatabaseManager.js]: Changed with commit a2aaad9

**Review by ghost — APPROVED** [review]

**Pull request merged**